mhs/common/setup.py

```diff
@@ -3,7 +3,7 @@
 import setuptools
 
 setup(
-    name='mhs_common',
+    name='common',
```

Comment:
I think the previous name (`mhs_common`) was better. Or `mhs-common` would be more consistent with the other Pipfile usages.